com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDAdditionalCameraData.cs

```diff
@@ -37,13 +37,24 @@ public class HDPhysicalCamera
         // Lens
         // Note: focalLength is already defined in the regular camera component
         [SerializeField][Range(kMinAperture, kMaxAperture)] float m_Aperture = 16f;
+        [SerializeField][Min(0f)] float m_FocusDistance = 10f;
```

Comment:
This is bounded differently vs https://github.com/Unity-Technologies/Graphics/pull/4286/files#diff-9ee958b0bde47c58ac148fcbc5dd458acb9a4e0fa124b2ebc3676cc515eae2a4R90 . I am not sure if there was a reason why the one in the volume was not 0, but maybe worth homogenize based on whatever of the two make sense?

Reply:
Good catch, in the end I kept the same minimum as the volume.